AlgebraInLean/World0LeanIntroSOLUTIONS/Sheet6Solutions.lean

```diff
@@ -0,0 +1,34 @@
+/-
+This is a solutions sheet.
+-/
+
+/- Tactics!
+
+Below is a list of tactics mentioned in World 0. Feel free to use this list for
+reference to write notes, or keep updating it with tactics mentioned in future worlds.
```

Comment:
This link might be worth linking to: [FM 2024 tactic cheatsheet](https://www.ma.imperial.ac.uk/~buzzard/xena/formalising-mathematics-2024/Part_C/cheatsheet.html)